### PR TITLE
NXO Demo: UI complete

### PR DESCRIPTION
**Pull request by jcnoriega**
Thank you for your contribution to PayPal. 

> Before submitting this PR, note that we cannot accept language translation PRs. We have a dedicated localization team to provide the translations. If there is an error in a specific translation, you may open an issue and we will escalate it to the localization team.

### Summary of changes

 - This PR completed the UI for Native Checkout Demo
 

 ### Checklist

 - [ ] Added a changelog entry

-

### CODE REVIEW

**Review by andrespelaezp — APPROVED** [review]

File: Demo/src/main/java/com/paypal/android/api/services/PayPalApi.kt [hunk 1]
```diff
@@ -18,6 +19,7 @@ interface PayPalApi {
     @Headers("Accept: application/json")
     @POST("/v2/vault/payment-tokens")
     suspend fun postApprovalSessionId(
+        @Header("Authorization") accessToken: String,
```
Comment (kgangineni) [inline Demo/src/main/java/com/paypal/android/api/services/PayPalApi.kt:22]: Q: why is this needed?

**Review by kgangineni — APPROVED** [review]
LGTM

File: Demo/src/main/java/com/paypal/android/api/services/PayPalApi.kt (hunk 1 above)
Comment (jcnoriega) [reply Demo/src/main/java/com/paypal/android/api/services/PayPalApi.kt:22]: `v2/vault/payment-tokens` requires and Access Token to work
